server/index.js

```diff
@@ -49,6 +49,8 @@ const scopes = [
   'delete:grants',
   'read:guardian_factors',
   'update:guardian_factors',
+  'read:mfa_policies',
```

Comment:
I'm basically blindly adding the scopes here, not sure I have to. Can the reviewer explain what these scopes here do and confirm whether I should add them here?

Reply:
These scopes are eventually passed to the `/authorize` URL back on the RTA when the extension is launched. So, if the extension needs to make any Management API calls that require these scopes, the scopes should be added here. 👍

Reply:
Thanks for the review @seejee !

> So, if the extension needs to make any Management API calls that require these scopes, the scopes should be added here

In that case, they were probably unnecessary. The extension itself is not reading/updating the mfa policies, I think it's only concerned with clients and resource servers. Although it's probably not critical that the scope is added here, it might have been better if the access token was granted only the minimal scopes necessary to perform its duty.